eslint.config.mjs

```diff
@@ -0,0 +1,127 @@
+import eslint from '@eslint/js'
+import vitest from '@vitest/eslint-plugin'
+import importPlugin from 'eslint-plugin-import'
+import jestDom from 'eslint-plugin-jest-dom'
+import react from 'eslint-plugin-react'
+import reactCompiler from 'eslint-plugin-react-compiler'
+import reactHooks from 'eslint-plugin-react-hooks'
+import testingLibrary from 'eslint-plugin-testing-library'
+import globals from 'globals'
+import tseslint from 'typescript-eslint'
+
+export default tseslint.config(
+  {
+    ignores: ['**/dist/'],
+  },
+  eslint.configs.recommended,
+  tseslint.configs.recommended,
+  react.configs.flat.recommended,
+  react.configs.flat['jsx-runtime'],
+  {
+    languageOptions: {
+      globals: {
+        ...globals.browser,
+        ...globals.node,
+        ...globals.es2015,
+      },
+      parser: tseslint.parser,
+      parserOptions: {
+        ecmaVersion: 2018,
+        sourceType: 'module',
+        ecmaFeatures: {
+          jsx: true,
+        },
+      },
+    },
+    plugins: {
+      import: importPlugin,
+      'react-compiler': reactCompiler,
+      'react-hooks': reactHooks,
+    },
+    settings: {
+      react: {
+        version: 'detect',
+      },
+      'import/extensions': ['.js', '.jsx', '.ts', '.tsx'],
+      'import/parsers': {
+        '@typescript-eslint/parser': ['.ts', '.tsx'],
+      },
+      'import/resolver': {
+        typescript: {
+          alwaysTryTypes: true,
+          project: 'tsconfig.json',
+        },
+      },
+    },
+    rules: {
+      eqeqeq: 'error',
+      curly: ['warn', 'multi-line', 'consistent'],
+      'no-console': 'off',
```

Comment:
```suggestion
      'no-console': 'warn',
```
@dai-shi Since `no-console` is not configured by `eslint.configs.recommended`, it is an unnecessary setting, so I was planning to remove it. However, if you want to avoid the use of `console.log` in the code, how about changing its setting `off` to `warn` to improve code quality?

Reply:
we allow `console.log`.

Reply:
Ok, i remove it